src/engraving/rw/compat/compatutils.cpp

```diff
@@ -339,6 +340,52 @@ void CompatUtils::reconstructTypeOfCustomDynamics(MasterScore* score)
     }
 }
 
+void CompatUtils::splitArticulations(MasterScore* score)
+{
+    std::vector<Articulation*> toRemove;
+    for (Measure* meas = score->firstMeasure(); meas; meas = meas->nextMeasure()) {
+        for (Segment& seg : meas->segments()) {
+            if (!seg.isChordRestType()) {
+                continue;
+            }
+            for (EngravingItem* item : seg.elist()) {
+                if (!item || !item->isChord()) {
+                    continue;
+                }
+                Chord* chord = toChord(item);
+                for (Articulation* a : chord->articulations()) {
```

Comment:
Don't we need to do something about linked items? (i.e. in linked staves and linked part scores). If so, then I think `toRemove` will need to be a `std::set`, like in the other methods in this file, because items in linked staves will be encountered twice.